setup.cfg

```diff
@@ -137,8 +137,3 @@ bandit.plugins =
 all_files = 1
 build-dir = doc/build
 source-dir = doc/source
-
-[pbr]
-autodoc_tree_index_modules = True
-autodoc_tree_excludes =
-  examples*
```

Comment:
I wasn't proposing to remove all dependencies on pbr, only the *runtime* dependency. If I understand correctly, Bandit also uses pbr as a build dependency both for the package and for the docs, and it can continue to do so.